src/Domain/Tags.hpp

```diff
@@ -122,6 +123,33 @@ struct ElementMap : db::SimpleTag {
   using type = ::ElementMap<VolumeDim, TargetFrame>;
 };
 
+/// \ingroup DataBoxTagsGroup
+/// \ingroup ComputationalDomainGroup
+/// Base tag to retrieve the centers of objects in the domain corresponding to
+/// the `ObjectLabel`.
+template <ObjectLabel Label>
+struct GridCenter : db::BaseTag {};
+
+/// \ingroup DataBoxTagsGroup
+/// \ingroup ComputationalDomainGroup
+/// The center of the excision sphere for the given object.
+///
+/// Even though this can easily be retrieved from the domain, we add it as its
+/// own tag so we can access it through the base tag. This way, other things
+/// (like the control system) can grab the center and be agnostic to what the
+/// object actually is.
+template <ObjectLabel Label>
+struct ExcisionGridCenter : GridCenter<Label>, db::SimpleTag {
+  using type = DataVector;
```

Comment:
Can we store this as `tnsr<double, Dim, Grid>`? DataVector is really not the right data type for this.

Reply:
Yes we can, but the control system needs a `DataVector` in order to do math with so I'll end up copying it then. Not really sure which is preferred.

Reply:
What sort of math? I'd rather use the right data type here and then make sure we can do the math with it that we need.

Reply:
So at the end of the day, I need a `DataVector` to update the functions of time. Currently, the AH centers come as `std::array` from the Strahlkorper which I convert to a `DataVector` before I insert it into the control system DataBox. So the most natural type for the grid centers was a `DataVector`. I have to be be able to add, subtract, dot, and cross these quantities.

So I guess I could make everything take a `tnsr` and do math with those and then copy it into a `DataVector` at the very end, but that requires a lot more changes and I think is beyond the scope of this PR.

Reply:
Actually I think everything you need is already handled by #4666. There are more places that could switch to `tnsr`, but as you say, that's beyond the scope of this change.

Reply:
Yes #4666 will make the tag a `tnsr`, but I'll still have to copy it into a `DataVector` inside the control error. Once this is merged I'll consider changing how we store the AH center to also be a `tnsr` so we can do math with a "proper" data type for what we are doing.